### PR TITLE
Source Monday: Update schema; expected_records

### PR DESCRIPTION
**Pull request by artem1205**
## What

Fix Source Monday: Connector health report

## How

- Fix schema;
- Fix expected records;

## Recommended reading order
1. `expected_records.jsonl`

## 🚨 User Impact 🚨

no breaking changes

## Pre-merge Actions

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-monday test report (commit [`b2c4af3647`](https://github.com/airbytehq/airbyte/pull/27501/commits/b2c4af36477b497728ca12ecf6f0f7d43bfb7168)) - ✅

⏲️  Total pipeline duration: 175 seconds

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-monday/metadata.yaml | ✅        |
| Connector version semver check.                                      | ✅        |
| Connector version increment check.                                   | ✅        |
| QA checks                                                            | ✅        |
| Code format checks                                                   | ✅        |
| Connector package install                                            | ✅        |
| Build source-monday docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                           | ✅        |
| Acceptance tests                                                     | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5323579867)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-monday test
```

File: airbyte-integrations/connectors/source-monday/Dockerfile [hunk 1]
```diff
@@ -34,5 +34,5 @@ COPY source_monday ./source_monday
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.2.6
```
Comment (davydov-d) [inline airbyte-integrations/connectors/source-monday/Dockerfile:37]: this should be a major version bump https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors

**Review by davydov-d — APPROVED** [review]

File: airbyte-integrations/connectors/source-monday/Dockerfile (hunk 1 above)
Comment (artem1205) [reply airbyte-integrations/connectors/source-monday/Dockerfile:37]: fixed

**Comment by octavia-squidington-iii** [comment]
##  source-monday test report (commit [`3177468568`](https://github.com/airbytehq/airbyte/pull/27501/commits/31774685680c976690786a07ae5eb1bf2040ad3f)) - ❌

⏲️  Total pipeline duration: 193 seconds

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-monday/metadata.yaml | ✅        |
| Connector version semver check.                                      | ✅        |
| Connector version increment check.                                   | ✅        |
| QA checks                                                            | ❌        |
| Code format checks                                                   | ✅        |
| Connector package install                                            | ✅        |
| Build source-monday docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                           | ✅        |
| Acceptance tests                                                     | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5327564966)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-monday test
```

**Comment by octavia-squidington-iii** [comment]
##  source-monday test report (commit [`5ff8bd6343`](https://github.com/airbytehq/airbyte/pull/27501/commits/5ff8bd6343ef32348f90ed4165bb21122944e674)) - ❌

⏲️  Total pipeline duration: 184 seconds

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-monday/metadata.yaml | ✅        |
| Connector version semver check.                                      | ✅        |
| Connector version increment check.                                   | ✅        |
| QA checks                                                            | ❌        |
| Code format checks                                                   | ✅        |
| Connector package install                                            | ✅        |
| Build source-monday docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                           | ✅        |
| Acceptance tests                                                     | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5327573226)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-monday test
```

**Comment by octavia-squidington-iii** [comment]
##  source-monday test report (commit [`d8b0f9e866`](https://github.com/airbytehq/airbyte/pull/27501/commits/d8b0f9e8669cb2ebfec01d7d4ae182dad9be235c)) - ❌

⏲️  Total pipeline duration: 146 seconds

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-monday/metadata.yaml | ✅        |
| Connector version semver check.                                      | ✅        |
| Connector version increment check.                                   | ✅        |
| QA checks                                                            | ❌        |
| Code format checks                                                   | ✅        |
| Connector package install                                            | ✅        |
| Build source-monday docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                           | ✅        |
| Acceptance tests                                                     | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5327702526)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-monday test
```

**Comment by octavia-squidington-iii** [comment]
##  source-monday test report (commit [`eb3e92da3a`](https://github.com/airbytehq/airbyte/pull/27501/commits/eb3e92da3aceb7a3105a52cab6c619ae2432b389)) - ✅

⏲️  Total pipeline duration: 436 seconds

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-monday/metadata.yaml | ✅        |
| Connector version semver check.                                      | ✅        |
| Connector version increment check.                                   | ✅        |
| QA checks                                                            | ✅        |
| Code format checks                                                   | ✅        |
| Connector package install                                            | ✅        |
| Build source-monday docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                           | ✅        |
| Acceptance tests                                                     | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5333221759)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-monday test
```

**Comment by octavia-squidington-iii** [comment]
##  source-monday test report (commit [`7a5ac2afb5`](https://github.com/airbytehq/airbyte/pull/27501/commits/7a5ac2afb5a976ec9f0765a14e078d2392d043a9)) - ❌

⏲️  Total pipeline duration: 505 seconds

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-monday/metadata.yaml | ✅        |
| Connector version semver check.                                      | ✅        |
| Connector version increment check.                                   | ✅        |
| QA checks                                                            | ❌        |
| Code format checks                                                   | ✅        |
| Connector package install                                            | ✅        |
| Build source-monday docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                           | ✅        |
| Acceptance tests                                                     | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5333665277)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-monday test
```

**Comment by artem1205** [comment]
> @artem1205 I believe you should start a breaking change release process: https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit#heading=h.r2xzw0kclodc

@alafanechere ,
Should we consider these changes as breaking, if we've not seen any issues related to this changes yet?

**Comment by alafanechere** [comment]
> Should we consider these changes as breaking, if we've not seen any issues related to this changes yet?
Changing a field type has implication on users destination data. Some columns previously typed as integer will be transformed to string, which can break the transformation they built on top of these tables.

**Comment by artem1205** [comment]
Closing, as will be fixed in https://github.com/airbytehq/airbyte/pull/27410

**Pull request closed without merging**